### ISSUE TITLE
Clean up translation files

### ISSUE THREAD

**Issue by ktohalloran** [azavea/echo-locator#548]
The translation files have grown as the app has evolved, adding new phrases but not removing old, now irrelevant ones. This task is to clean up those files and remove phrases that are no longer necessary.

**Comment by ddohler** [comment]
@ktohalloran can this be closed or are we waiting on final approval from BHA?

**Comment by ktohalloran** [comment]
@ddohler This is more of a tech debt card to clean up translations from the files that are no longer needed, as those files are unwieldy and contain a lot of language from previous iterations of the app. The final edits to translations for this phase of the project were finished in card #568 so we don't need anything more from BHA on that front.

**Comment by ddohler** [comment]
Got it, thanks, I put a technical debt label on this to clarify!
